fe2-android/app/src/androidTest/java/com/github/dedis/popstellar/pages/qrcode/ElectionStartPageObject.java

```diff
@@ -0,0 +1,34 @@
+package com.github.dedis.popstellar.pages.qrcode;
```

Comment:
I think this should be placed in `com.github.dedis.popstellar.pages.detail.event.consensus`